app/services/own_tracks/params.rb

```
@@ -90,4 +90,37 @@ def owntracks_point?
+      when 'charging' then 2
+      when 'full' then 3
+      else 0
+      end
+    end
+
+    def connection_to_string(connection)
+      case connection
+      when 'mobile' then 'm'
+      when 'wifi' then 'w'
+      when 'offline' then 'o'
+      else nil
```

RuboCop (1.81.7)

[convention] Redundant `else`-clause.

(Style/EmptyElse)